doc/ref_arch/openstack/chapters/chapter02.md

```diff
@@ -48,18 +49,42 @@ Proposals (WK): Change title of this section to 2.3 Architectural Requirements;
-| `req.inf.ntw.05` | Network | The Architecture **shall** support Distributed Virtual Routing (DVR) to allow compute nodes to route traffic efficiently |
-| `req.inf.acc.01` | Acceleration | The Architecture **shall** support Application Specific Acceleration (exposed to VNFs) xxx. |
-| `req.inf.acc.02` | Acceleration | The Architecture **shall** support NFVI Acceleration (such as SmartNICs) |
-| `req.inf.acc.03` | Acceleration | The Architecture **shall not** rely on SR-IOV PCI-Pass through to provide acceleration to VNFs |
+| `req.inf.stg.03` | Storage | The Architecture **should** be able to accommodate VNFs that store back into its image through use of hypervisor attached volumes. |
+| `req.inf.stg.04` | Storage | The Architecture **should** make the immutable images available via location independent means. |
+| `req.inf.stg.05` | Storage | The Architecture **should** provide high-performance and horizontally scalable VIM instance storage |
+| `req.inf.stg.06` | Storage | The Architecture **should** make shared storage available to tenants as network-based File or Object Storage and support very large file needs. |
+| `req.inf.stg.07` | Storage | The Architecture **should** allow to use externally provided large archival storage for its Backup / Restore / Archives needs. |
+| `req.inf.stg.08` | Storage | The Architecture **should** make available all non-host OS / Hypervisor / Host systems storage as network-based Block, File or Object Storage for tenant/management consumption. |
+| `req.inf.ntw.01` | Network | The Architecture **must** provide virtual network interfaces to VM instances |
+| `req.inf.ntw.02` | Network | The Architecture **must** include an SDN integration to support provisioning of network services from host-based OpenStack Neutron networking VTEPs to the Border Edge based VRFs. |
+| `req.inf.ntw.03` | Network | The Architecture **must** Localize intra-host traffic to provide low latency, high throughput, and resiliency. |
+| `req.inf.ntw.04` | Network | The Architecture **should** support service function chaining. |
+| `req.inf.ntw.05` | Network | The Architecture **must** allow for East/West tenant traffic within the cloud (via tunnelled encapsulation overlay - VXLAN or Geneve). |
+| `req.inf.ntw.06` | Network | The Architecture **should** support Distributed Virtual Routing (DVR) to allow compute nodes to route traffic efficiently |
+| `req.inf.ntw.07` | Network | The Architecture **must** provide low latency and high throughput (do we need to quantify it?). |
+| `req.inf.ntw.08` | Network | The NFVI Network Fabric **should** embrace the concepts of open networking and disaggregation using commodity networking hardware and disaggregated Network Operating Systems |
+| `req.inf.ntw.09` | Network | The NFVI Network Fabric **should** embrace open-based standards and technologies. |
+| `req.inf.ntw.10` | Network | The NFVI Network Fabric **must** be capable of supporting carrier-specific VNF workloads. |
```

Comment:
This requirement is too unspecific, especially "carrier-specific" is undefined. Proposal to delete.

Reply:
I actually disagree.  This is very important requirement (which actually was used in real life implementation).  We can be more specific, and use carrier-grade VNF workloads, which I hope everybody will understand.

Reply:
In my understanding  the whole exercise is about VNF workloads, therefore the requirement does not add anything new. "carrier-grade" is just a buzz-word used by vendors -- the question is, what are the concrete requirements. If possible we should here refer to concrete metrics the infrastructure has to deliver.